pycqed/measurement/multi_qubit_module.py

```diff
@@ -3689,7 +3691,8 @@ def measure_n_qubit_rabi(qubits, sweep_points=None, amps=None, prep_params=None,
 
     if exp_metadata is None:
         exp_metadata = {}
-    exp_metadata.update({'preparation_params': prep_params,
+    exp_metadata.update({'qubit_names': qubit_names,
```

Comment:
@stephlazar 
- Why are the names stored in metadata for this measurement, but not for the other multiqubit time-domain measurements?
- Why are they called qubits_names in the metadata instead of qb_names, which is the usual name in analysis classes?